frontend/lib/features/forms/SchemaForms.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_form_builder/flutter_form_builder.dart';
+import 'dart:convert';
+
+
+// example schema: 
+    String jsonSchema = r'''
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "properties": {
+    "accountNumber": {
+      "type": "string",
+      "description": "Mobile Money account number of the Recipient",
```

Comment:
All this schema would be cool to see in a widget test. You could load it up and `expect` to see all the fields etc. You can also load in sample data to verify you get the expected errors.

Reply:
added a test

Reply:
Since we have a test now, we can probably delete this file and just reintroduce the code here when needed.